### ISSUE TITLE
Missing item drop from Overlord Wyrmthalak

### ISSUE THREAD

**Issue by flurrz** [LightsHope/server#721]
Item in question: https://web.archive.org/web/20050427225330/http://wow.allakhazam.com:80/db/item.html?witem=13148

**Comment by QAkvastskaft** [comment]
The following 2 items does not currently drop:

https://web.archive.org/web/20060524043515/http://wow.allakhazam.com:80/db/item.html?witem=13148
https://web.archive.org/web/20060612215754/http://wow.allakhazam.com:80/db/item.html?witem=13164

They were removed sometime during vanilla as seen by the almost non-existent drop-rate for later vanilla.
Early vanilla: https://web.archive.org/web/20060302090331/http://wow.allakhazam.com:80/db/mob.html?wmob=9568
Late vanilla: https://web.archive.org/web/20060831063510/http://wow.allakhazam.com:80/db/mob.html?wmob=9568

According to comment here http://db.vanillagaming.org/?item=13164#wh-comments
The trinket was replaced with trinket 22321 which by its item id was added in the item overhaul in 1.10
so both items were likely removed/replaced the same patch

In summary:
item id 13148 and 13164 should drop from mob id 9568 until 1.10
in patch 1.10 item 13148 and 13164 should be removed and item 22321 should be added to mob id 9568
